subgraphs/abracadabra/src/common/tokens.ts

```diff
@@ -76,108 +76,103 @@ export function fetchTokenDecimals(tokenAddress: Address): i32 {
   }
 
   // try with the static definition
-  let staticTokenDefinition = StaticTokenDefinition.fromAddress(tokenAddress);
+  let staticTokenDefinition = fromAddress(tokenAddress);
   if (staticTokenDefinition != null) {
     return staticTokenDefinition.decimals as i32;
-  } else {
-    return INVALID_TOKEN_DECIMALS as i32;
   }
+
+  return INVALID_TOKEN_DECIMALS as i32;
 }
 
 export function isNullEthValue(value: string): boolean {
-  return value == "0x0000000000000000000000000000000000000000000000000000000000000001";
+  if (value == "0x0000000000000000000000000000000000000001" || value == "0x0000000000000000000000000000000000000000")
```

Comment:
Why `….0001` is considered null?

Reply:
It's a burn address: https://etherscan.io/address/0x0000000000000000000000000000000000000001